sql/src/main/java/org/apache/druid/sql/http/SqlQuery.java

```diff
@@ -66,7 +66,7 @@ public SqlQuery(
   )
   {
     this.query = Preconditions.checkNotNull(query, "query");
-    this.resultFormat = resultFormat == null ? ResultFormat.OBJECT : resultFormat;
+    this.resultFormat = resultFormat == null ? ResultFormat.DEFAULT_RESULT_FORMAT : resultFormat;
```

Comment:
Why is this change needed ?

Reply:
This is the default format when nothing is specified at all, but it is only specified in this class. It would be better if this was clearly specified so that it could be reused (or changed in the future), so I added `DEFAULT_RESULT_FORMAT`